istio/istio.go

```diff
@@ -439,13 +444,16 @@ func (iClient *IstioClient) ApplyOperation(ctx context.Context, arReq *meshes.Ap
 				opName = "removed"
 			}
 			iClient.eventChan <- &meshes.EventsResponse{
-				EventType: meshes.EventType_INFO,
-				Summary:   fmt.Sprintf("Book Info app %s successfully", opName),
-				Details:   fmt.Sprintf("The Istio canonical Book Info app is now %s.", opName),
+				OperationId: arReq.OperationId,
+				EventType:   meshes.EventType_INFO,
+				Summary:     fmt.Sprintf("Book Info app %s successfully", opName),
+				Details:     fmt.Sprintf("The Istio canonical Book Info app is now %s.", opName),
```

Comment:
Will we have a function specific to each sample app?

Reply:
Not necessarily. Only the ones which need extra care probably have a separate method.

Reply:
Good. Ideally, not. Assuming we can generic-ize, so should we keep the message/string generic where possible.